src/styles/blocks/main.scss

```diff
@@ -0,0 +1,64 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+body {
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  display: flex;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: wrap;
+  gap: $gapWidth;
+  padding: $paddingWidth;
+
+  width: calc($dayWidth * 7 + $paddingWidth * 2 + $gapWidth * 6);
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: $dayWidth;
+    height: $dayHeight;
+    border: 1px solid;
+    background-color: #eee;
+
+    &:hover {
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      cursor: pointer;
+      transition-duration: 0.5s;
+    }
+  }
```

Comment:
The animation is choppy -
https://gyazo.com/d735cce7cf5d37b90df40f648b707349

How should it be -
https://gyazo.com/01ccfd37dd2fae2bae29abc07bddd3dc

This can be fixed like this -

```suggestion
    border: 1px solid;
    background-color: #eee;
    transition: transform 0.5s;

    &:hover {
      background-color: #ffbfcb;
      transform: translateY(-20px);
      cursor: pointer;
    }
  }
```